sw/device/tock/boards/opentitan/tock_dependencies/Cargo.toml

```diff
@@ -0,0 +1 @@
+/home/svt/development/repos/opentitan/sw/device/tock/boards/opentitan/tock_dependencies/Cargo_remote.toml
```

Comment:
You need to update the `.gitignore` to ignore this file.

Reply:
Good point, thanks.

Reply:
Thanks for mentioning this.

Reply:
`Cargo_remote.lock` should **not** be ignored as this is how we pin the build to upstream tock revisions. In general the `.gitignore` should be updated to reflect the new paths.